src/components/Avatar/README.md

```diff
@@ -18,8 +18,10 @@ A component for rendering a circular container used to display a user's initials
 
 ## API
 
```

Comment:
...Whole file...

I wonder if this README should serve both `Avatar` and `AvatarGroup`. Maybe not... but maybe we can at least link to `AvatarGroup`s README from here.